libs/vkd3d/command.c

```diff
@@ -2209,16 +2231,19 @@ static void d3d12_command_list_clear_attachment_pass(struct d3d12_command_list *
     begin_info.clearValueCount = clear_op ? 1 : 0;
     begin_info.pClearValues = clear_op ? clear_value : NULL;
 
-    VK_CALL(vkCmdBeginRenderPass(list->vk_command_buffer,
-            &begin_info, VK_SUBPASS_CONTENTS_INLINE));
+    VK_CALL(vkCmdBeginRenderPass2KHR(list->vk_command_buffer,
```

Comment:
Do we actually need to use BeginRenderPass2 as well?

Reply:
I guess not but I don't see why we'd mix and match stuff here. Any Mesa driver will just wrap it for us.

Reply:
Fair enough I guess.